### PR TITLE
Provide prettier README and INSTALL

### PR DESCRIPTION
**Pull request by nicoonoclaste**
As some Markdown-like notation was already used (though not very consistently)
in `README`, I thought it would be a good idea to prettify it.

This need to be proof-read, though, as some changes were automated using `sed`.
While I checked the diff before commiting, I may have overlooked something.

### CODE REVIEW

**Comment by hrj** [comment]
Any plans to merge this @greenrd ?

**Comment by aspiers** [comment]
Big :+1: from me for this.  Is `topgit` still maintained?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,785 @@
...
+branches easier. In fact, it is designed especially for the case where
+you maintain a queue of third-party patches on top of another (perhaps
+Git-controlled) project and want to easily organize, maintain and submit
+them - TopGit achieves that by keeping a separate topic branch for each
+patch and providing some tools to maintain the branches.
+
+
+Installation
+============
+See the file [`INSTALL`](INSTALL).
```
Comment (aspiers) [inline README.md]: Shouldn't this be `INSTALL.md`?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,785 @@
...
+   patch queue, deferring to StGIT instead might make more sense.
+
+2. When using TopGit, your history can get a little hairy
+   over time, especially with all the merges rippling through. ;-)
+
+
+Synopsis
+========
+
+## Create and evolve a topic branch
```
Comment (aspiers) [inline README.md]: I think these should be at the `h3` level, i.e. `###`.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,785 @@
+TopGit - A different patch queue manager
+
+
+Description
+===========
```
Comment (aspiers) [inline README.md]: Better to use `-` not `=`?  I think `=` is more commonly used for the document title, not for section titles.

**Comment by greenrd** [comment]
This version of TopGit is now unmaintained - please see https://github.com/mackyle/topgit for a newer version, which has a prettier README.

**Pull request closed without merging**